pkg/releaser/git.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	"github.com/google/go-github/github"
```

Comment:
I think this is a rather old version of `go-github`. To get a more recent version, you can use
`"github.com/google/go-github/v29/github"`

Reply:
from the go-github documentation, it seems v29 is recommended when using go-modules. 

(behind the scenes I believe code is same).

and i am actually running behind on adoption of go-modules. I will catch up on that and move my repositories to use that version.